src/Microsoft.VisualStudio.VsInteractiveWindow/source.extension.vsixmanifest

```diff
@@ -12,15 +12,6 @@
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
-  <Installer>
-    <Actions>
-      <Action Type="Ngen" Path="Microsoft.VisualStudio.InteractiveWindow.dll" />
-      <Action Type="Ngen" Path="Microsoft.VisualStudio.VsInteractiveWindow.dll" />
-    </Actions>
-  </Installer>
-  <Dependencies>
```

Comment:
If that is the case, I would suggest fixing up the InstallationTarget on line 10-13 to be limited to installing on Dev15+